lib/modules/auth/screens/login.screen.dart

```diff
@@ -10,7 +10,7 @@ class LoginScreen extends StatelessWidget {
   Widget build(BuildContext context) {
     return Scaffold(
```

Comment:
This overflows when opening the keyboard; either stop the Scaffold from resizing with 
```suggestion
    return Scaffold(
      resizeToAvoidBottomInset: false,
```
Or wrap the Login() with a SingleChildScrollView (cannot add this as a suggestion as there's a "deleted" line), which will allow for scrolling when the keyboard is open

Reply:
Can't test stuff on mobile because I don't have the proper Android Studio setup yet (tested in browser). We should probably overwrite that in the Theme if it is a global expected behavior.

Reply:
Now that I think about it, I believe using the SingleChildScrollView is the better choice; I don't think we're supposed to add tree structure-related configurations to the theme, and this behavior should be decided for each screen because it depends on whether or not we care about parts of the screen being hidden by the keyboard. In this case, the buttons should be accessible even if you have the keyboard open

Reply:
Wrapped it but I can't test it. Should just merge because I can't work on this anymore.